docs/specs/code/SERVER_STATUS_ENDPOINT.md

```diff
@@ -0,0 +1,37 @@
+# Abstract
+
+Describe a new endpoint to notify server issues in Yoroi-frontend.
+   
+# Motivation
+
+Currently, Yoroi-frontend has no way to notify neither server issues nor connection issues. Also, once Yoroi has the ability to notify server issues, the endpoint used by Yoroi to fetch the information can be used to create a public URL that reports the mentioned status, such as [AWS](https://status.aws.amazon.com/) and [Twitter](https://status.twitterstat.us).
+
+# Proposal
+
+Adding a new endpoint in the backend named `api/status` is proposed. This endpoint has the method `GET` that returns the last JSON inserted in the following table.
+
+```
+CREATE TABLE api_statuses (
+   ID serial NOT NULL PRIMARY KEY,
+   status json NOT NULL
+);
+```
+
+The element insertion will be performed manually into the database.
+
+The members of this JSON are TBD. Tentatively and as stated by [ch1223 conversation](https://app.clubhouse.io/emurgo/story/1223/healthcheck-endpoint), the JSON contains a boolean with the state value, a string with the problem description and a timestamp.
+
+The string with the problem description can be used to be displayed in a banner that Yoroi-frontend will render. However, this comes with the problem of i18n, i.e., showing an English message in all the languages or inserting the problem description translated for all the available languages in the JSON. Considering this, it is preferred to display the same message regardless of the problem description (and have this message translated in `app/i18n`).
```

Comment:
we could add i18n to the JSON if we need to add something custom on the fly

Reply:
I'm not convinced hunting people down to get translations generated when our server is having issues is really worth it. I would rather have a fixed message. This is also safer because if we allow custom messages, then if somebody can take over our server they can send malicious messages to users such as what happened in the Electrum case (https://www.theblockcrypto.com/tiny/hackers-used-a-clever-tactic-to-steal-over-700k-from-users-of-a-popular-bitcoin-wallet/)

Reply:
@SebastienGllmt 

- We don’t need to hunt people, but if we want to put a specific message for English and Japanese we will be able to do it. The other way we never will be able to. One solution would contain the other one.
- Yeah, I remember that hack, but what I commented is not related to that because it’s just multi-text for multi-language if we want to. They were rendering html from the server 🤦‍♂ 
- If we want an extra level of security, we can sign the messages in the DB with a small group of keys that we use to sign our commits. Yoroi could verify the validity of the signing of those messages, this way even if our servers get hack, it wouldn’t be possible to falsify our signatures - same would apply for man-in-the-middle attacks.